src/app/content/highlights/components/cardUtils.ts

```diff
@@ -71,3 +71,12 @@ export const generateUpdatePayload = (
 
   return {updatePayload, preUpdateData};
 };
+
+export const noopKeyCombinationHandler = (activeElement: Element | null): boolean => {
+  if (!activeElement) { return false; }
+  if (activeElement.nodeName === 'TEXTAREA') { return true; }
+  if (activeElement.nodeName === 'INPUT') {
```

Comment:
i feel as if there should be a way to check if an element accepts keyboard input without whitelisting them... but i can't find anything online.

Reply:
I think that we shouldn't disable `useKeyCombination` on every editable element because we could use it with another keyboard shortcut that uses for example `ctrlKey` so it can be used while editing
`useKeyCombination` is expecting _optional_ `noopHandler` function so it can be customized like in this case